docs/sphinx/source/solvers.rst.in

```diff
@@ -0,0 +1,65 @@
+.. Note that this file is processed into solvers.rst before being included in the sphinx build
+
+####################
+Solvers and examples
+####################
+
+Building NESO with configuration option ``-DNESO_BUILD_SOLVERS=ON`` will generate a number of solver executables that can be used to run example simulations.
+Note that this option is turned on by default, but is disabled when building NESO via spack with the ``+libonly`` variant.
+
+Source code for each solver is in ``solvers/<solver_name>`` and input files for the solver examples can be found in ``examples/<solver_name>``.
+Note that these examples are intended to demonstrate various features of the NESO library, rather than to simulate particular scenarios of interest in plasma physics.
+The solver source codes also act as potential starting points for constructing new applications.
+
+More general instructions for :ref:`running solvers<run_eg>` and :ref:`modifying example meshes<geo_to_xml>` can be found towards the end of this section. 
+The following subsections describe the solvers and accompanying examples that are currently available in NESO.
+
+
+.. include:: solver_readmes.md
+   :parser: myst_parser.sphinx_
+
+
+.. _run_eg:
+
+****************
+Running a solver
+****************
+
+While each solver executable can be used directly, the simplest way to run a solver example is using the ``run_eg.sh`` script.
+This ensures all required input files are copied to a new directory (``runs/<solver_name>/<examples_name>``) and that the correct command is executed.
+To run the script from the repository root, use:
+
+   <RUN_EG_USAGE>
+
+Note that the number of MPI processes can be set via a command line flag, and defaults to 4.
+The script assumes the intended ``mpirun`` executable is on your path and looks for solver executables in the most recently modified spack "view" directory.
+To use a different solver executable, supply the build directory or install directory as described.
```

Comment:
Is a small section here on common footguns, e.g. not setting `OMP_NUM_THREADS`, useful?

Reply:
Yes, good point; will add that

Reply:
Something like this?
![image](https://github.com/user-attachments/assets/d139d556-5f43-4491-94b5-1556605e6d87)

Or I can add a proper subsection if you can think of other warnings that should go here?

Reply:
Pushed that change